src/openms/include/OpenMS/ANALYSIS/TARGETED/TargetedExperimentHelper.h

```diff
@@ -84,6 +85,23 @@ namespace OpenMS
 
     };
 
+    /// @brief This class stores the nucleic acid equivalent of a protein, compare with nuctide, which is the nucleic acid peptide
+    struct Oligo :
+      public CVTermList
+    {
+      Oligo() = default;
```

Comment:
probably better to remove (rule of zero)

Reply:
This is copied from the Protein struct above. I can circle back after I've gotten it working as well as the original.